Formula/m/morpheus.rb

```diff
@@ -0,0 +1,98 @@
+class Morpheus < Formula
+  desc "Modelling environment for multi-cellular systems biology"
+  homepage "https://morpheus.gitlab.io/"
+  url "https://gitlab.com/morpheus.lab/morpheus/-/archive/v2.3.7/morpheus-v2.3.7.tar.gz"
+  sha256 "ad5694a098e4752db53659ee983c3ae417a43747320e73c3005f6cf88b52d55c"
+  license "BSD-3-Clause"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+(?:_?\d+)?)$/i)
+  end
+
+  depends_on "boost" => :build
+  depends_on "cmake" => :build
+  depends_on "doxygen" => :build
+  depends_on "ninja" => :build
+  depends_on "ffmpeg"
+  depends_on "graphviz"
+  depends_on "libomp"
+  depends_on "libtiff"
+  depends_on "qt@5"
+
+  uses_from_macos "bzip2"
+  uses_from_macos "libxml2"
+  uses_from_macos "zlib"
+
+  def install
+    # has to build with Ninja until: https://gitlab.kitware.com/cmake/cmake/-/issues/25142
+    args = []
+    args << "-G Ninja"
+
+    if OS.mac?
+      args << "-DMORPHEUS_RELEASE_BUNDLE=ON"
+      args << "-DMORPHEUS_SBML=OFF" # SBML import currently disabled due to libSBML build errors with some macOS SDKs
+    end
+
+    system "cmake", "-S", ".", "-B", "build", *args, *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+
+    if OS.mac?
+      bin.write_exec_script "#{prefix}/Morpheus.app/Contents/MacOS/morpheus"
+      bin.write_exec_script "#{prefix}/Morpheus.app/Contents/MacOS/morpheus-gui"
+    end
+  end
+
+  def post_install
+    if OS.mac? && File.read("#{prefix}/Morpheus.app/Contents/Info.plist").include?("HOMEBREW_BIN_PATH")
+      # Set PATH environment variable including Homebrew prefix in macOS app bundle
+      inreplace "#{prefix}/Morpheus.app/Contents/Info.plist", "HOMEBREW_BIN_PATH", "#{HOMEBREW_PREFIX}/bin"
+    end
+  end
+
+  def caveats
+    on_macos do
+      <<~EOS
+        To start the Morpheus GUI, type the following command:
+
+          morpheus-gui
+
+        Or add Morpheus to your Applications folder with:
+
+          ln -sf #{opt_prefix}/Morpheus.app /Applications
+      EOS
+    end
+  end
```

Comment:
```suggestion
```
This isn't really Homebrew-specific (for the former), and the latter tends to confuse users (e.g. symlinking app bundles doesn't work with Spotlight)

Reply:
That's a good point. Unfortunately, the app bundle indeed doesn't show up in Spotlight this way. Do you have a suggestion on how this could be achieved?

However, the symlink is important for many users in that the app will appear in the Launchpad.

In addition to the command-line tool `morpheus`, `morpheus-gui` adds important functionality whose existence and usage should be pointed out to the user in the caveats in my opinion. Furthermore, the launch of `morpheus-gui` is relevant for the registration of the URI scheme `morpheus://...`.

Reply:
Distributing GUI applications really isn't a goal in Homebrew/core so if this is required it might not be a good fit to add this to Homebrew/core.

Reply:
> Do you have a suggestion on how this could be achieved?

No, there is no good way for achieving this. That's why we avoid telling users about `.app` bundles installed by formulae, because it just doesn't result in a good user experience. (Users can still find it for themselves, but they're on their own when they do that.)